contrib/mesos/pkg/minion/server.go

```diff
@@ -164,7 +164,7 @@ func (ms *MinionServer) launchProxyServer() {
 // executor doesn't support failover right now, the right thing to do is to fail completely since all
 // pods will be lost upon restart and we want mesos to recover the resources from them.
 func (ms *MinionServer) launchExecutorServer(containerID string) <-chan struct{} {
-	allArgs := os.Args[1:]
+	allArgs := os.Args[2:]
```

Comment:
related to a recent fix that went in, sorry should have xref'd it. our
scheduler used to build poorly formed commandInfo.Arguments. I fixed it,
which introduced an extra arg at the beginning. which means that we should
ignore the additional arg here (read: this wants to ignore the "km"
"minion" args from the command line)

On Fri, Mar 4, 2016 at 11:01 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In contrib/mesos/pkg/minion/server.go
> <https://github.com/kubernetes/kubernetes/pull/22500#discussion_r55047580>
> :
>
> > @@ -164,7 +164,7 @@ func (ms *MinionServer) launchProxyServer() {
> >  // executor doesn't support failover right now, the right thing to do is to fail completely since all
> >  // pods will be lost upon restart and we want mesos to recover the resources from them.
> >  func (ms *MinionServer) launchExecutorServer(containerID string) <-chan struct{} {
> > -	allArgs := os.Args[1:]
> > +	allArgs := os.Args[2:]
>
> Oh, the opposite in fact.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/22500/files#r55047580>.
>